portality/static/js/doaj.fieldrender.edges.js

```diff
@@ -2065,8 +2065,18 @@ $.extend(true, doaj, {
 
             this.namespace = "doaj-public-search";
 
+            this.selector = edges.getParam(params.selector, null)
+            this.currentQueryString  = "";
+
+
             this.draw = function () {
-                var frag = "<li class='alert'><p><strong>No results found</strong> that match your search.</p> <p>Try removing some of the filters you have set, modifying the text in the search box, or using less specific search terms.</p></li>";
+                if (doaj.publicSearch.activeEdges[this.selector]){
```

Comment:
I'm not a big fan of this, the inner components shouldn't need to know about the outer layers of Edges, and this prevents this component being used anywhere else as it is now tightly bound to the `doaj.publicSearch`.  If what you want is to get at the parent Edge, then use:

```
this.component.edge
```

Reply:
fixed